sdk/include/opentelemetry/sdk/metrics/meter_context.h

```diff
@@ -91,23 +109,34 @@ class MeterContext
                std::unique_ptr<View> view) noexcept;
 
   /**
-   * Force all active Exporters and Readers to flush any buffered meter data
+   * Adds a meter to the list of configured meters.
+   *
+   * Note: This method is not thread safe, and should ideally be called from main thread in
+   * thread-safe manner.
+   *
+   * @param meter
+   */
+  void AddMeter(std::shared_ptr<Meter> meter);
```

Comment:
Could `AddMeter` be implemented as thread-safe? Document this requirement in comment makes it less observable.

Reply:
Good point. It's supposed to be an internal method and called from MeterProvider::GetMeter() in a thread-safe manner. I will document it as internal so as to avoid any explicit usage.